lib/general-calendar/proper-of-time.ts

```diff
@@ -170,15 +170,15 @@ export class ProperOfTime extends CalendarDef {
 
     // The Sunday within the Octave of the Nativity of the Lord,
     // Feast of the Holy Family of Jesus, Mary and Joseph.
-    this.#newLiturgicalDayDef(`holy_family`, {
+    this.#newLiturgicalDayDef(`holy_family_of_jesus_mary_and_joseph`, {
       precedence: Precedences.GeneralLordFeast_5,
       dateDef: { dateFn: 'holyFamily', yearOffset: -1 + yearOffset },
-      isHolyDayOfObligation: false,
+      isHolyDayOfObligation: dow === 0,
```

Comment:
@emagnier, I have just noticed that this does not work as I expected, as `dow` is not defined here at all.

How could I get the `holy_family` date? I think putting `const dateDef = holyFamily()` and `const dow = getWeekNumber(dateDef)` before L173? Both `holyFamily()` and `getWeekNumber()` would be imported from `lib/utils/dates.ts`.

Note that same problem is with `baptismOfTheLord`.

Reply:
Indeed, I missed that when reviewing the PR.

In fact, you don't have to take care here of the `dow === 0`, because `isHolyDayOfObligation` is automatically changed to `true` if this liturgical day falls in a Sunday.